keyboards/ergodox/keymaps/narze/keymap.c

```diff
@@ -0,0 +1,852 @@
+#define HPR_ESC ALL_T(KC_ESC)
+#define SFT_ENT SFT_T(KC_ENT)
+#define SFT_PO F(M_SFT_PO)
+#define SFT_PC F(M_SFT_PC)
+#define GUI_MINS GUI_T(KC_MINS)
+#define GUI_UNDS F(M_GUI_UNDS)
+
+// Combo : SuperDuper layer from S+D (R+S in Colemak)
+// #define COMBO_COUNT 1
+// #define SUPERDUPER_COMBO_COUNT 3
+// #define EECONFIG_SUPERDUPER_INDEX (uint8_t *) 19
+
+// enum process_combo_event {
+//   CB_SUPERDUPER,
+// };
+
+// const uint16_t PROGMEM superduper_combos[SUPERDUPER_COMBO_COUNT][3] = {
+//   [_QWERTY] = {KC_S, KC_D, COMBO_END},
+//   [_COLEMAK] = {KC_R, KC_S, COMBO_END},
+//   [_QWOC] = {CM_S, CM_D, COMBO_END},
+// };
+
+// combo_t PROGMEM key_combos[COMBO_COUNT] = {
+//   [CB_SUPERDUPER] = COMBO_ACTION(superduper_combos[_QWERTY]),
+// };
+
+// volatile bool superduper_enabled = true;
+
+// const uint16_t empty_combo[] = {COMBO_END};
+
+// void set_superduper_key_combos(void);
+// void clear_superduper_key_combos(void);
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+/* Qwerty
+ *
+ * ,--------------------------------------------------.           ,--------------------------------------------------.
+ * |   `    |   1  |   2  |   3  |   4  |   5  | Mdia |           | Mdia |   6  |   7  |   8  |   9  |   0  |   -    |
+ * |--------+------+------+------+------+-------------|           |------+------+------+------+------+------+--------|
+ * | Tab    |   Q  |   W  |   E  |   R  |   T  | Symb |           | Symb |   Y  |   U  |   I  |   O  |   P  |   \    |
+ * |--------+------+------+------+------+------|      |           |      |------+------+------+------+------+--------|
+ * | Hp/Esc |   A  |   S  |   D  |   F  |   G  |------|           |------|   H  |   J  |   K  |   L  |   ;  |   '    |
+ * |--------+------+------+------+------+------| Hyper|           | Meh  |------+------+------+------+------+--------|
+ * | Sft/(  | Z/Mo |   X  |   C  |   V  |   B  |      |           |      |   N  |   M  |   ,  |   .  | SD-/ | Sft/)  |
+ * `--------+------+------+------+------+-------------'           `-------------+------+------+------+------+--------'
+ *   |Rse/[ | Ctrl |  Alt | Gui/_|  Lwr |                                       |  Rse | GUI/-|  Alt | Ctrl | Low/]  |
+ *   `----------------------------------'                                       `----------------------------------'
+ *                                        ,-------------.       ,-------------.
+ *                                        | Enter| LGui |       | Alt  |Ctrl/Esc|
+ *                                 ,------|------|------|       |------+--------+------.
+ *                                 |      |      | Home |       | PgUp |        |      |
+ *                                 | Space| Back |------|       |------| Back   |Enter |
+ *                                 |      | Space| End  |       | PgDn | Space  |      |
+ *                                 `--------------------'       `----------------------'
+ */
+[_QWERTY] = KEYMAP(
```

Comment:
We're phasing out `KEYMAP` in favor of `LAYOUT`.  If you could change this? 

```suggestion
[_QWERTY] = LAYOUT_ergodox(
```